src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -1480,6 +1502,9 @@ def setup(self):
             dtype=self.dtype,
         )
 
+    def get_shared_layer(self):
```

Comment:
Is this method strictly necessary? One can access the shared layers through `model.shared` (or something to that effect!) and the params through `model.params`